src/main.c

```diff
@@ -688,12 +688,12 @@ int main(int argc, char* argv[]) {
       // Parse error
       jv msg = jv_invalid_get_msg(value);
       if (!(options & SEQ)) {
-        // --seq -> errors are not fatal
```

Comment:
Confusing, comment was wrong?

Reply:
Comment was placed in misleading place so I moved it.